dev/wlp.lib.extract/resources/wlp/lib/extract/SelfExtractMessages.nlsprops

```diff
@@ -276,6 +276,11 @@ UNABLE_TO_FIND_TOOLS_JAR=The Liberty Java agent could not be attached because th
 
 UNABLE_TO_FIND_JAVA_AGENT=The Liberty Java agent could not be located. This server will run, but it cannot perform functions that require the Java agent, such as monitoring or tracing.
 
+RUN_IN_CHILD_JVM_DEBUG=WLP_JAR_DEBUG was set. The server will run in a separate Java virtual machine.
```

Comment:
How about..."You set the WLP_JAR_DEBUG property. The server now runs in a separate virtual machine."